wiki/osu!wiki_Contribution_Guide/en.md

```diff
@@ -0,0 +1,72 @@
+
+*Side note: please don't use a fake or a throwaway email account. You will recieve emails regarding pull request reviews and such in the future.*
+
+### Forking
+
+To make changes, you will need to fork the `osu-wiki` repository.
+
+![](img/fork.jpg)
+
+Go to your fork (click the `Fork` button again). Seeing something similar to the image below means you made a fork of `ppy/osu-wiki` repo.
+
+![](img/forked.jpg)
+
+## Editing online or locally
+
+From this point, you have two choices:
+
+- [Online](/wiki/owcg/Online) - uses GitHub's web interface; this is best for single article edits
+- [Locally](/wiki/owcg/Locally) - uses GitHub Desktop; this is best for single and/or multi article edits (this includes uploading, deleting, and moving images or files)
+
+**When you have completed either Online or Locally, you can continue to the next section.**
+
+## Finishing
+
+### Opening a pull request
+
+Go to the [`ppy/osu-wiki` repo](https://github.com/ppy/osu-wiki); if you were quick enough, you may see this yellow banner. If you see it, click on the `Compare & pull request` button. If not, click the `New pull request` button.
+
+![](img/github-recent.jpg)
+
+On the next page, if you see these two buttons as pictured below, click the `compare across forks`.
+
+![](img/compare-across-forks-no.jpg)
+
+![](img/compare-across-forks-yes.jpg "This is what you want to see.")
+
+Click on the `head fork` dropdown and select the one with your username (it should be the second one).
+
+![](img/head-fork.jpg)
+
+Click on the `compare` dropdown and select the one with the branch you had created (these are listed alphabetically).
+
+![](img/compare-branch.jpg)
+
+Then click on the `Create pull request` button.
+
+![](img/create-pull-request.jpg)
+
+Enter the title and fill in the description box. Once you are ready, click the `Create pull request` button.
+
+![](img/new-pull-request.jpg)
+
+### Reviews
+
+Once you have created your pull request, other osu!wiki editors may review your changes to help catch some mistakes you may have missed. **You will need to keep up with these reviews,** otherwise your pull request may be marked for closure! If you want someone to review your pull request, you can ask other osu!wiki editors in the [osu!dev Discord chat](https://discord.gg/ppy) or in the GitHub comments.
+
+### Merging
+
+For your changes to become live, your pull request has to be merged. Once your pull request has been reviewed, you can either use the commenting section in GitHub to ask someone to merge it or do the same in the [osu!dev Discord chat](https://discord.gg/ppy).
+
+Once this is completed, your changes are officially part of the osu!wiki. Your changes will not show up right away due to caching on the server's end, but they will show up within one to five hours.
```

Comment:
it's better to add a new section about repo sync using https://upriver.github.io

Reply:
I'll put something like that in the common mistakes page... I'll rename that page to common issues or something too

Reply:
but they aren't really best practices either... the current (and my unpushed local) is pretty much the workarounds of using the GitHub web interface